Cooper/ClientApp/src.v2/app/components/avatar-card/avatar-card.component.scss

```diff
@@ -1,10 +1,9 @@
 .avatar-card {   
     object-fit: cover;
     width: 230px;
-    height: auto;
+    height: 280px;
```

Comment:
Refactoring that file is not related to issue. It overlaps the changes that were made in #173 (Nickname bug on profile) pull-request